explorer/client/src/components/cookies-consent/analytics.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import ReactGA from 'react-ga4';
+ReactGA.initialize('G-96DM59YK2F');
+ReactGA.send('pageview');
```

Comment:
What should we do if we want to collect more fine-grained events, e.g., collect event for `object view`, `transaction view`, `click on transaction details`, etc?

Reply:
We can will add other event logs. Just started with default page view